### DISCUSSION TITLE
The YOLO achievement could be perceived as negative

### DISCUSSION THREAD

**Discussion by eli-front** [community/community#18375] [Other Feature Feedback, Questions, & Ideas] [answered]
<img width="1440" alt="Screen Shot 2022-06-11 at 3 06 27 PM" src="https://user-images.githubusercontent.com/63875354/173203292-ce10dfcf-eec7-4480-a7cf-f84af0669e58.png">

The YOLO achievement is for when you merge a PR without a review. This is pure semantics, but I think this may infer a lack of effort/responsibility. For me, I don't believe this will show on my public profile because it's from a personal private repo, but I'd be interested to hear your thoughts.

**Comment by verduscos** [comment]
I think we've all done this at one point or another....I don't see it as negative.

**Reply by amirjani** [reply]
I see it the same way

**Reply by samuelfmlourenco** [reply]
I do it constantly, when I merge my own pull requests on my projects. Why would I need to review my own PR?

**Reply by efimovrusl** [reply]
You need to add a reviewer, so solo projects don't fall under this

**Reply by IDKHowToDev** [reply]
i got it just because i changed the readme without reviewing,now im afraid no one will hire me

**Comment by alkock** [comment]
The best thing about Git is the versioning. You can't destroy much. Of course, pushing it into the main branch is not necessarily the best way to behave. 
Otherwise, I can only agree with @verduscos.

**Comment by fwcd** [comment, marked as answer]
As of https://github.blog/changelog/2022-10-07-visibility-settings-for-individual-achievements/, individual achievements can now be hidden, which should fix this issue.

**Comment by serohant** [comment]
I think so too
